homeassistant/components/emulated_hue/hue_api.py

```diff
@@ -205,12 +212,6 @@ def put(self, request, username, entity_number):
             # as the actual requested command.
             config.cached_states[entity_id] = (result, brightness)
 
-        # Separate call to turn on needed
```

Comment:
That was my suggestion since the separate on/brightness was not terribly useful for real devices. It also alters the semantics for the test.